find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
```

Comment:
`Array is empty` might be a confusing message to print out since a user can call the method on an array that isn't empty, but set the max index to zero or negative. 
```suggestion
if max_index == 0
    puts "Array has no elements in the given range"
    return nil
else if max_index < 0
   puts "Invalid range: Max index can't be less than zero"
   return nil
```